### ISSUE TITLE
Read LOCAL OPML files

### ISSUE THREAD

**Issue by RafaelLinux** [imelgrat/opml-parser#4]
I'm trying to read a OPML file (sources.opml) stored in same path that opml-parser.php  in server, but it doesn't work, even preceding with "_DIR_" the path.

```
$O_Contenido->ParseLocation('sources.opml', null);    // doesn't work
$O_Contenido->ParseLocation('./sources.opml', null);    // doesn't work
$O_Contenido->ParseLocation(__DIR__.'/sources.opml', null);    // doesn't work
$O_Contenido->ParseLocation('https:/mysite/sources.opml', null); // works
```

Am I doing something wrong? I noticed that you use "curl" to retrieve file, but not sure if that's the issue.

Thank you

**Comment by imelgrat** [comment]
Hi @RafaelLinux. If you want to use this class on a local file, you should use the **file://** protocol, followed by the full path of the file you're trying to read. The file protocol is disabled by cURL if open_basedir is set. 

Best regards,
Iván

**Comment by RafaelLinux** [comment]
file:// on  server? As I wrote, the file will be read by PHP as a file IN server folder.

**Comment by imelgrat** [comment]
Yes. The file:// protocol is used for reading files stored in a server folder.
